src/libraries/System.Diagnostics.TraceSource/src/System/Diagnostics/SwitchLevelAttribute.cs

```diff
@@ -7,7 +7,7 @@ namespace System.Diagnostics
     [AttributeUsage(AttributeTargets.Class)]
     public sealed class SwitchLevelAttribute : Attribute
     {
-        private Type _type;
+        private Type _type = null!;
```

Comment:
Make ctor assign to field directly perhaps?

Reply:
Same here, we can add with null check too if needed